lib/tasks/wiremock.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'net/http'
+
+namespace :wiremock do
```

Comment:
I guess we can get rid of this if we lean on the docker container?

Reply:
I think its quite handy to be able to run this locally (outside of docker), e.g. for capturing new requests